src/api/app/views/webui2/webui/project/_tabs.html.haml

```diff
@@ -10,26 +10,29 @@
           = tab_link('Maintained Projects', project_maintained_projects_path)
       - unless project.defines_remote_instance? || project.is_maintenance?
         %li.nav-item
-          = tab_link('Repositories', repositories_path)
+          = tab_link('Repositories', repositories_path(project))
       %li.nav-item
-        = tab_link('Requests', project_requests_path)
+        = tab_link('Requests', project_requests_path(project))
       - unless project.defines_remote_instance?
         %li.nav-item
-          = tab_link('Users', project_users_path)
+          = tab_link('Users', project_users_path(project))
+      - if controller_name == 'staging_workflows'
+        %li.nav-item
+          = tab_link('Staging', staging_workflow_path(project.staging))
```

Comment:
so this tab only appears when you're actually on the tab?

Reply:
It looks like :rofl: It should be `if project.staging` instead of `if controller_name == 'staging_workflows'`

Reply:
> so this tab only appears when you're actually on the tab?

Currently yes, we don't want this tab visible at least you are in a staging workflow

Reply:
what about a comment above the `if`? it is not something evident :pray: